wiki/osu!_wiki/en.md

```diff
@@ -0,0 +1,41 @@
+# osu! wiki
+
+The **osu! wiki** is an open-source knowledge database written and maintained by [volunteer contributors](https://github.com/ppy/osu-wiki/graphs/contributors). It contains a wide range of information on topics related to osu!, its community, and related activities and concepts. The wiki serves as an official source of information and is considered as a documentation of the osu! history.
```

Comment:
```suggestion
The **osu! wiki** is an open-source knowledge database written and maintained by [volunteer contributors](https://github.com/ppy/osu-wiki/graphs/contributors). It contains a wide range of information on topics related to osu!, its community, and related activities and concepts. The wiki serves as an official source of information and is considered as a documentation of osu! history.
```

Reply:
odd question, but why?

Reply:
just style/may sound better (for instance "the history" on its own may sound incomplete in sentence); could otherwise write "the history of osu!"